assets/admin_themes/Sea_Green/fileManager/fileManager.scss

```diff
@@ -0,0 +1,308 @@
+.file-manager {
+  #browseiframe {
+    border: 1px solid $g-900 !important;
+  }
+}
+
+.file-manager-body {
+  all: unset;
+  background: $white !important;
+  color: $g-900 !important;
+  box-shadow: $boxshadow-small !important;
+  .ri-delete-bin-line {
+    color: $error !important;
+  }
+
+  .ui-dialog {
+    border: 1px solid $g-900 !important;
+    box-shadow: none !important;
+    .ui-dialog-titlebar {
+      box-shadow: none;
+      border: 1px solid $g-900;
+      background: none;
+      color: $g-900;
+      text-shadow: none;
+      border-radius: 0;
+    }
+    .ui-dialog-content {
+      box-shadow: none;
+      .uniform-input {
+        background: none;
+        border: 1px solid $g-900;
+        box-shadow: none;
+        &:focus {
+          box-shadow: 4px 4px 0px $elevation-shadow !important;
+        }
+      }
+    }
+
+    .ui-dialog-buttonpane {
+      box-shadow: none;
+      button {
+        background: none;
+        box-shadow: none;
+        border: 1px solid $g-900;
+        padding: 4px 10px;
+        &:hover {
+          text-shadow: none !important;
+          box-shadow: $btn-hover-boxshadow !important;
+        }
+        .ui-button-icon {
+          margin: auto;
+        }
+      }
+    }
+    .box {
+      box-shadow: none;
+      border-radius: 0;
+    }
+
+    .ui-dialog-titlebar-close {
+      display: none;
+    }
+    .about div.head a {
+      background: none;
+    }
+  }
+
+  #menu {
+    .ui-menu {
+      box-shadow: none !important;
```

Comment:
Lots of `important` going on. Is that needed even when the selector is made more precise than the previous "default" CSS selectors? Dunno which is best.